documentation/latest/diagrams/workload_deployment.puml

```diff
@@ -9,14 +9,14 @@ actor User
 User -> "EdgeWorkload CR": Create
 Operator -> "EdgeWorkload CR": Process
 Operator -> "EdgeDevice CR": add workload reference to status
-Operator -> Agent: Send pod yaml
+"Edge API" -> Agent: Send pod yaml
```

Comment:
the direction of arrow *4* should be opposite, and the text might need to be changed as well, so it will be clear that this is one-way communication (the flotta doesn't send, it is the agent that pulls)

Reply:
Done.